osu.Game/Overlays/Mods/ModSelectOverlay.cs

```diff
@@ -228,60 +232,50 @@ public ModSelectOverlay()
-                                                    Right = 20
-                                                }
+                                                Origin = Anchor.CentreLeft,
+                                                Anchor = Anchor.CentreLeft,
                                             },
-                                            new OsuSpriteText
+                                            new FillFlowContainer
                                             {
-                                                Text = @"Score Multiplier:",
-                                                Font = OsuFont.GetFont(size: 30),
-                                                Margin = new MarginPadding
-                                                {
-                                                    Top = 5,
-                                                    Right = 10
-                                                }
-                                            },
-                                            MultiplierLabel = new OsuSpriteText
-                                            {
-                                                Font = OsuFont.GetFont(size: 30, weight: FontWeight.Bold),
-                                                Margin = new MarginPadding
+                                                AutoSizeAxes = Axes.Both,
+                                                Spacing = new Vector2(footer_button_spacing / 2, 0),
+                                                Origin = Anchor.CentreLeft,
+                                                Anchor = Anchor.CentreLeft,
+                                                Children = new Drawable[]
                                                 {
-                                                    Top = 5
-                                                }
+                                                    new OsuSpriteText
+                                                    {
+                                                        Text = @"Score Multiplier:",
+                                                        Font = OsuFont.GetFont(size: 30),
+                                                        Origin = Anchor.CentreLeft,
+                                                        Anchor = Anchor.CentreLeft,
+                                                    },
+                                                    MultiplierLabel = new OsuSpriteText
+                                                    {
+                                                        Font = OsuFont.GetFont(size: 30, weight: FontWeight.Bold),
+                                                        Origin = Anchor.CentreLeft,
+                                                        Anchor = Anchor.CentreLeft,
+                                                        Width = 70, // to avoid footer from flowing when clicking mods
```

Comment:
for slightly better English:
```suggestion
                                                        Width = 70, // to prevent footer from flowing when clicking mods
```